### DISCUSSION TITLE
You are trying to animate y from "0px"

### DISCUSSION THREAD

**Discussion by PaulStat** [heroui-inc/heroui#2382] [Help]
Hello, I am expecting this issue when trying to use modal in my Next.js application. this warning appears each time modal popups. 
Anyway it works, modal popups, maybe with little flickering...  But each time drops this warning
![Screenshot 2024-02-19 150748](https://github.com/nextui-org/nextui/assets/155636462/1dd5072e-9ca8-43b4-a18c-6362df2f0068)
 in console.
You are trying to animate y from "0px" to "var(--slide-enter))". 0px is not an animatable value - to enable this animation set 0px to a value animatable to var(--slide-enter)) via the `style` property.

What can cause that?

**Comment by PaulStat** [comment]
I found a problem, it is a small bug in library: it is left by mistake double brackets at the end of : y: "var(--slide-enter))", 
(it is in modal component index.js file. :
`var scaleInOut = {
  enter: {
    scale: "var(--scale-enter)",
    y: "var(--slide-enter))",
    opacity: 1,
    transition: {
      scale: {
        duration: 0.4,
        ease: import_framer_transitions.TRANSITION_EASINGS.ease
      },
      opacity: {
        duration: 0.4,
        ease: import_framer_transitions.TRANSITION_EASINGS.ease
      },
      y: {
        type: "spring",
        bounce: 0,
        duration: 0.6
      }
    }
  },
  exit: {
    scale: "var(--scale-exit)",
    y: "var(--slide-exit)",
    opacity: 0,
    transition: {
      duration: 0.3,
      ease: import_framer_transitions.TRANSITION_EASINGS.ease
    }
  }
};
`
As solution I I have added a motionProps to Modal to rewrite original motion params:
`motionProps={{
            variants: {
              enter: {
                scale: "var(--scale-enter)",
                y: "var(--slide-enter)",
                opacity: 1,
                transition: {
                  scale: {
                    duration: 0.4,
                    ease: [0.36, 0.66, 0.4, 1],
                  },
                  opacity: {
                    duration: 0.4,
                    ease: [0.36, 0.66, 0.4, 1],
                  },
                  y: {
                    type: "spring",
                    bounce: 0,
                    duration: 0.6,
                  },
                },
              },
              exit: {
                scale: "var(--scale-exit)",
                y: "var(--slide-exit)",
                opacity: 0,
                transition: {
                  duration: 0.3,
                  ease: [0.36, 0.66, 0.4, 1],
                },
              },
            },
          }}`

**Reply by J4v4Scr1pt** [reply]
Nice find dude, I also found this issue, but there is an easier fix. Update to 2.2.10 and it's fixed :).
Maybe you did a PR..
